src/styles/product-card.css

```diff
@@ -0,0 +1,107 @@
+html {
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+}
+
+.product-card {
+  display: flex;
+  flex-direction: column;
+
+  width: 200px;
+  padding: 32px 16px 16px;
+  box-sizing: border-box;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  background: var(--color-block-background);
+}
+
+.product-card__image {
+  width: 160px;
+  height: 134px;
+
+  margin: 0 auto 40px;
+  object-fit: contain;
+}
+
+.product-card__title {
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  color: var(--color-text-primary);
+
+  margin-bottom: 4px;
+
+}
+
+.product-card__code {
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--color-text-secondary);
+
+  margin-bottom: 16px;
+}
+
+.product-card__rating {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+
+  line-height: 14px;
+
+  margin-bottom: 24px;
+}
+
+.product-card__reviews {
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--color-text-primary);
+}
+
+.product-card__price {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+
+  margin-bottom: 16px;
+}
+
+.product-card__price-name {
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--color-text-secondary);
+}
+
+.product-card__price-amount {
+  font-size: 16px;
+  font-weight: 700;
+  line-height: 18px;
+  color: var(--color-text-primary);
+}
+
+.product-card__buy {
+  width: 166px;
+  height: 40px;
```

Comment:
Better to set needed paddings instead of height hardcoding, probably we'll need other height for button if we'll change it's text font-size